stepprogress/src/main/java/com/funrisestudio/stepprogress/StepProgressView.kt

```diff
@@ -505,6 +507,42 @@ class StepProgressView : ViewGroup, View.OnClickListener {
         return stepProgress.isAllDone()
     }
 
+    fun setTextTitlePadding(@DimenRes size: Int) {
+        textTitlePadding = resources.getDimension(size)
+        resetView()
```

Comment:
resetView method recreates views responsible for showing nodes. Here we have only layout changes for title position so we don't need to call resetView(). After invalidate() new dimensions will be applied

Reply:
Did you check method behaviour without resetView()? It seems that it is not needed there. You can also run requestLayout() to remeasure views

Reply:
fixed